doc/ref/states/top.rst

```diff
@@ -64,304 +134,224 @@ just the state tree, then the :conf_master:`file_roots` option can be expanded:
       prod:
         - /srv/salt/prod
 
-Then our top file could reference the environments:
+In the above, we declare three environments: `dev`, `qa` and `prod`.
+Each environment has a single directory assigned to it.
+
+Our top file references the environments:
 
 .. code-block:: yaml
 
     dev:
-      'webserver*dev*':
+      'webserver*':
```

Comment:
These hostnames with the env contained within the ID were probably intentional. If you don't pin a minion to an env with the methods you mention below in "Choosing an Environment" then the environment is chosen from the top file. The result of this change means the `webserver*` minions are now in all the dev, qa, and prod environments!